LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -1,5 +1,10 @@
 module LibTrixi
 
+using Reexport: @reexport
+
+@reexport using OrdinaryDiffEq
+@reexport using Trixi
```

Comment:
We have discussed this for TrixiShallowWater.jl before, and concluded that it is probably _not_ a good idea to reexport everything from those packages. Why? Because whenever there is a breaking change in those packages, we also need to do a breaking change in LibTrixi.jl. Thus I'd recommend to just write
```suggestion
using OrdinaryDiffEq
using Trixi
```
or similar...

Reply:
Sounds reasonable!

How do I access e.g. `LinearScalarAdvectionEquation1D` in the libelixir then?

Reply:
Good point. We could either a) have them exposed through `using LibTrixi.Trixi` or b) add them directly to the project file in `libtrixi-julia`. I think b) is the more flexible option, since that will allow us to directly control exactly which version of Trixi.jl and OrdinaryDiffEq.jl is used.

Reply:
Still unsure about this: should we always start the libelixir files with
`using Trixi, OrdinaryDiffEq`
or can this be done somewhere else, s.t. one can still write e.g. `LinearScalarAdvectionEquation1D` in the libelixir?

Reply:
TBH, I wouldn't do it anywhere else. The advantage of having it in the libelixir itself is that it makes it more flexible (we could also add additional packages that we don't even know of yet. Also, if the packages are included, it makes it a proper Julia file in its own right, i.e., something one can run in an arbitrary Julia session for testing etc.

Reply:
I resolved this by adding Trixi.jl and OrdinaryDiffEq.jl as explicit dependencies to the project directory in 799859c. That way, one can just write
```julia
using LibTrixi
using OrdinaryDiffEq
using Trixi
```
inside the libelixirs and be done with it. That allows one also to easily switch out the Trixi.jl package for a local installation/dev'd version if desired.